### ISSUE TITLE
Need to provide the sample data to go with a "production" run

### ISSUE THREAD

**Issue by karllark** [BEAST-Fitting/beast#135]
The workflow branch uses METAL data as part of the production example.  Need to change this to using published PHAT data so that it can be distributed.

**Comment by cemurray3** [comment]
I am currently working on this! PR #196 is a start. Currently, I am fiddling with AST files to go along with a full brick (15) example PHAT dataset for an example production run.

**Comment by cmurray-astro** [comment]
1.5 years later I am working on this again! In the beast-examples repository, @christinawlindberg and I are working on including a cutout of M31-B21-WEST (new PHAT photometry of Brick 21) with a Jupyter Notebook for helpful explanation.

**Comment by cmurray-astro** [comment]
*Update!*

In collaboration with @christinawlindberg, I cut out a tiny "chunk" of Brick 9 East (not 21 West as stated above) from PHAT (M31-B09-EAST). ASTs already exist for the whole brick, but are not dense enough to sample the source density bins within the cutout field, so assigned them all random positions within the cutout field (i.e., dumped them all in). The "chunk" photometry catalog, AST catalog, and reference image, as well as the script I used to create them (via Jupyter Notebook; wrangling_data.ipynb) can all be found in this dropbox folder: https://www.dropbox.com/sh/91aefrp9gzdc9z0/AAC9Gc4KIRIB520g6a0uLLama?dl=0

I wonder if these data are too big or would be useful for the workflow / tests?

**Comment by karllark** [comment]
Not at all.  These are all smaller than files we use for the current BEAST regression tests.
